draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -411,42 +411,45 @@ extension, that can improve the precision of the feedback from QUIC is
 {{!I-D.draft-ietf-quic-ack-frequency}}, which allows a sender to control the
 delay of acknowledgments sent by the receiver.
 
+The list of RTCP Receiver Reports that could be replaced by feedback from QUIC follows:
+
 * *Receiver Reports* (`PT=201`, `Name=RR`, {{!RFC3550}})
-  * *Fraction lost*: The fraction of lost packets can be directly infered from
+  * *Fraction lost*: When RTP packets are carried in QUIC datagrams, the fraction of lost packets can be directly inferred from
     QUIC's acknowledgments. The calculation SHOULD include all packets up to the
     acknowledged RTP packet with the highest RTP sequence number. Later packets
     SHOULD be ignored, since they may still be in flight, unless other QUIC
-    packets that were sent after the datagram frame, were already acknowledged.
+    packets that were sent after the RTP packet frame, were already acknowledged.
   * *Cumulative lost*: Similar to the fraction of lost packets, the cumulative
-    loss can be infered from QUIC's acknowledgments including all packets up to
+    loss can be inferred from QUIC's acknowledgments including all packets up to
     the latest acknowledged packet.
   * *Highest Sequence Number received*: The highest sequence number received is
-    the sequence number of all RTP packets that were acknowledged.
-  * Interarrival jitter: If QUIC acknowledgments carry timestamps as described
+    the highest sequence number of all RTP packets carried in a QUIC packet that was acknowledged.
```

Comment:
In another PR I added some more text to explain what the highest sequence number is in RTP:
```
  * Highest Sequence Number received*: In RTCP, this field is a 32-bit field
    that contains the highest sequence number a receiver received in an RTP
    packet and the count of sequence number cycles the receiver has observed. A
    sender sends RTP packets in QUIC packets and receives acknowledgments for
    the QUIC packets. By keeping a mapping from a QUIC packet to the RTP packets
    encapsulated in that QUIC packet, the sender can infer the highest sequence
    number and number of cycles seen by the receiver from QUIC acknowledgments.
```

Does it make sense to add this explanation, and if yes, should we add it to this PR?

Reply:
I think we should add this explanation. (Whether it is done here or in a separate PR, authors can decide that.)

Reply:
@mengelbart, I agree with @goelvidhi about adding this text. I put it in #60, so if it's also in another PR (which I haven't looked for yet), that should also be fine.